frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -1729,7 +1733,11 @@ export async function getTimeTrackingForUserSummedPerAnnotation(
   if (annotationTypes != null) params.append("annotationTypes", annotationTypes);
   if (projectIds != null && projectIds.length > 0)
     params.append("projectIds", projectIds.join(","));
-  params.append("annotationStates", "Active,Finished");
+  if (annotationState !== "All") {
```

Comment:
Yeah, I also tried this one, but this means that a non type / full import of `AnnotationStateFilterEnum` is needed, which creates a new cyclic dependency :see_no_evil:.

The only think I could do it move the definition of `AnnotationStateFilterEnum` to some other file like `constants.ts`. What do you think?

Reply:
Yes, I'd move it to constants then and use it here. Strictly speaking constants shouldn't contain any types, but that ship has sailed :sweat_smile: This can be cleaned up later, together with the other types in there.